test/tests/functional/pbs_multi_sched.py

```diff
@@ -543,8 +545,7 @@ def test_job_sorted_per_scheduler(self):
 
     def test_qrun_job(self):
         """
-        Test jobs are sorted as per job_sort_formula
-        inside each scheduler
+        Test jobs can be run by qrun bypassing the scheduler.
```

Comment:
Are we bypassing the scheduler?  A normal qrun has the scheduler run the job.  You only bypass the scheduler with a qrun -H

Reply:
updated the test doc.